definitions/procedures/foreman/corrupted_roles.rb

```diff
@@ -0,0 +1,102 @@
+module Procedures::Foreman
+  class CorruptedRoles < ForemanMaintain::Procedure
```

Comment:
Could this be called FixCorruptedRoles or SplitCorruptedRoles or to see what it does?